.github/workflows/check.yml

```diff
@@ -32,13 +31,15 @@ jobs:
       - name: Configure build
         run: |
           chown -R builder. .
-          sudo -u builder -- ./autogen.sh
 
       - name: Build
-        run: sudo -u builder -- make --load-average=2
+        run: sudo -u builder -- tito buld --test --rpm
```

Comment:
There’s really no point in building things twice if you’re not storing the artifacts anywhere. Using Packit would solve the issue (and builds would happen in parallel and for multiple Fedora releases).

Reply:
This should be the primary way of building. I put the other 'make' in `run test` so we can have the artifacts.

Reply:
I’m talking about the resulting RPMs.

Reply:
There’s also a typo in in that line.